app/utils/answersHelpers/consignment/houseConsignment/ConsignmentItemAnswersHelper.scala

```diff
@@ -68,22 +69,28 @@ class ConsignmentItemAnswersHelper(
     prefix = "unloadingFindings.rowHeadings.item.countryOfDestination"
   )
 
-  def grossWeightRow: Option[SummaryListRow] = getAnswerAndBuildRow[BigDecimal](
+  def grossWeightRow: SummaryListRow = getAnswerAndBuildRowWithRemove[BigDecimal](
     page = GrossWeightPage(houseConsignmentIndex, itemIndex),
     formatAnswer = formatAsWeight,
     prefix = "unloadingFindings.rowHeadings.item.grossWeight",
     args = itemIndex.display,
-    id = Some(s"change-gross-weight-${houseConsignmentIndex.display}"),
-    call = Some(Call(GET, "#"))
+    id = s"gross-weight-${houseConsignmentIndex.display}",
```

Comment:
Will these not be duplicated across all items? Should this not be the itemIndex? Same for other methods

Reply:
Not sure on this, I don't think I added this index. It was the houseConsignment one before.